tests/components/accuweather/__init__.py

```diff
@@ -28,8 +31,8 @@ async def init_integration(
         options=options,
     )
 
-    current = json.loads(load_fixture("accuweather/current_conditions_data.json"))
-    forecast = json.loads(load_fixture("accuweather/forecast_data.json"))
+    current = load_json_object_fixture("accuweather/current_conditions_data.json")
```

Comment:
The reason we do the json parsing afterwards, is that if the test messes with the object, the fixture will not be updated. Because of the LRU cache, these changes will now remain between tests.

Reply:
Most fixtures won't be changed, and that's great. It's just a gotcha that maybe we should at least document (although no one will ever read it…)

Reply:
If that's a worry, we could also remove the LRU cache on `load_json***` and keep it only on `load_fixture`

Reply:
Yes I don't think we should cache fixture objects in a common helper. It's ok to cache a fixture string, but mutable objects should cached by each integration as wanted as that's fragile.

Reply:
Done - but I have kept this in draft until #88247 is merged to avoid conflicts